tokens/base/shadow/dark.json5

```diff
@@ -0,0 +1,74 @@
+{
+  "base": {
+    "shadow": {
+      "color": {
+        "small": {
+          "$value": "{base.color.scale.black}",
+          "alpha": 0.96,
+          "$type": "color"
+        },
+        "medium": {
+          "$value": "{base.color.scale.gray.4}",
+          "alpha": 0.85,
+          "$type": "color"
+        },
+        "large": {
+          "$value": "{base.color.scale.gray.4}",
+          "alpha": 0.8,
+          "$type": "color"
+        },
+        "extraLarge": {
+          "$value": "{base.color.scale.gray.4}",
+          "alpha": 0.7,
+          "$type": "color"
+        },
+        "highlight": {
+          "$value": "{base.color.scale.white}",
+          "alpha": 0.25,
+          "$type": "color"
+        },
+        "inset": {
+          "$value": "{base.color.scale.gray.2}",
+          "alpha": 0,
+          "$type": "color"
+        }
+      },
```

Comment:
I think it would be better to keep it with the shadows. Effectively this only adds an alpha value to an existing color.

Splitting it up would make working with shadows very complex, as you effectively define a shadow across two files per color mode (I think this gets to complex).

I this case I think the "structurally right" place is not worth the bad DX.